pkgs/development/libraries/libva/default.nix

```diff
@@ -61,5 +61,9 @@ stdenv.mkDerivation (finalAttrs: {
       "libva-glx" "libva-wayland" "libva-x11"
     ];
     platforms = platforms.unix;
+    badPlatforms = [
+      # Mandatory libva shared library.
+      lib.systems.inspect.platformPatterns.isStatic
```

Comment:
Is there a difference to stdenv.isStatic?

Reply:
Yes, that's a boolean based on the host system, so it would be suitable for `meta.broken`, but meta.platforms (and by extension badPlatforms) is supposed to be static metadata, so this is a pattern that always evaluates to the same thing, and is then checked by comparison to the current platform by the platform check.